rocky/rocky/templates/partials/list_filters.html

```diff
@@ -11,6 +11,13 @@
                 data-hide-filters-label="{% block hide_text %}{% translate "Hide filter options" %}{% endblock hide_text %} ">
             {% translate "Show filter options" %}
         </button>
+        {% if request.GET.sorting_options or request.GET.filter_options %}
+            <div class="notification">
+                <p>
+                    <span class="icon ti-circle-check"></span><strong>{% translate "Filters are on" %}</strong>
```

Comment:
From which figma design is the "Filters are on" coming from? I don't see it in the linked designs. This will also add it to all pages that use this partial.